framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/AbpDbContext.cs

```diff
@@ -298,6 +314,7 @@ private void PublishEventsForTrackedEntity(EntityEntry entry)
             case EntityState.Added:
                 ApplyAbpConceptsForAddedEntity(entry);
                 EntityChangeEventHelper.PublishEntityCreatedEvent(entry.Entity);
+                BulkEntityChangeEventHelper.PublishBulkEntityCreatedEvent(new List<object> { entry.Entity });
```

Comment:
Maybe move this to `SaveChangesAsync` to be able to handle bulk events properly ?